homeassistant/components/openrgb/__init__.py

```diff
@@ -0,0 +1,213 @@
+"""The OpenRGB integration."""
+import asyncio
+import logging
+
+from openrgb import OpenRGBClient
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT
+from homeassistant.const import CONF_CLIENT_ID, CONF_HOST, CONF_PORT
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.event import async_track_time_interval
+
+from .const import (
+    DEFAULT_CLIENT_ID,
+    DEFAULT_PORT,
+    DOMAIN,
+    ENTRY_IS_SETUP,
+    ORGB_DATA,
+    ORGB_DISCOVERY_NEW,
+    ORGB_TRACKER,
+    SERVICE_FORCE_UPDATE,
+    SERVICE_PULL_DEVICES,
+    SIGNAL_DELETE_ENTITY,
+    SIGNAL_UPDATE_ENTITY,
+    TRACK_INTERVAL,
+)
+from .helpers import orgb_entity_id
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
New integrations aren't allowed to support YAML config so this should not be defined.